.pre-commit-config.yaml

```diff
@@ -188,3 +188,7 @@ repos:
         entry: 'pg8000'
         files: ^ci/deps
         types: [yaml]
+    -   id: validate-min-versions-in-sync
```

Comment:
looks like this hook only runs on certain files? if so, you can put `files: `, so it doesn't need to be run if only unrelated files are modified

Reply:
Good call. Added the files that should be checked.